src/components/collapsibleCard/view/collapsibleCardView.js

```diff
@@ -1,7 +1,6 @@
 import React, { PureComponent } from 'react';
 import { View, TouchableHighlight } from 'react-native';
-import Animated, { EasingNode } from 'react-native-reanimated';
-
+import { View as AnimatedView } from 'react-native-animatable';
```

Comment:
@aliseyalvi please revert this...

Reply:
@noumantahir changes done https://github.com/ecency/ecency-mobile/pull/2530/commits/09a930489b5ae19f7d1b7eb84b5800e8cc67b80f